content/intro-to-storybook/react/en/deploy.md

```diff
@@ -19,7 +19,7 @@ This tutorial uses <a href="https://www.chromatic.com/">Chromatic</a>, a free pu
 
 ### Setup a repository in GitHub
```

Comment:
- "Setup" = noun. e.g. "Follow the setup process"
- "Set up" = verb, e.g. this heading

I noticed this multiple other places (worth a Find for "setup"), but only just now commented because we correctly use the verb form in the paragraph under this heading, in which we use the noun form incorrectly... so it's rather noticeable.

Reply:
Same for "login" vs. "log in".